mm2src/coins/utxo/qtum.rs

```diff
@@ -1112,7 +1112,7 @@ impl HDWalletBalanceOps for QtumCoin {
 
     async fn all_known_addresses_balances(
         &self,
-        hd_account: &HDCoinHDAccount<Self>,
+        hd_account: &UtxoHDAccount,
     ) -> BalanceResult<Vec<HDAddressBalance<Self::BalanceObject>>> {
         utxo_common::all_known_addresses_balances(self, hd_account).await
     }
```

Comment:
two lines under, `known_address_balance` still uses `HDBalanceAddress<Self>`.
https://github.com/KomodoPlatform/komodo-defi-framework/blob/eb991060409e4fd86157e7790c850272fda5a896/mm2src/coins/utxo/qtum.rs#L1120

Reply:
Thanks for noticing this, will push a fix asap.